src/globals.ts

```diff
@@ -434,6 +434,7 @@ export const SupportedLocales = Object.freeze([
   { name: 'Nederlands', code: 'nl' },
   { name: 'Português', code: 'pt' },
   { name: 'Русский', code: 'ru' },
+  { name: 'slovenščina', code: 'si' },
```

Comment:
[According to ISO 639-1](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=sl), the Slovenian identifier should be "sl" and not "si"

```suggestion
  { name: 'slovenščina', code: 'sl' },
```

Reply:
Thank you. I will change this and also rename locale file from si.yaml to sl.yaml